deploy/base/db/deployment.yaml

```diff
@@ -0,0 +1,92 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: mysql
+spec:
+  strategy:
+    type: Recreate
+  selector:
+    matchLabels:
+      app: mysql
+  template:
+    metadata:
+      labels:
+        app: mysql
+    spec:
+      serviceAccountName: mysql
+      securityContext:
+        fsGroup: 999
+        runAsUser: 999
+      restartPolicy: Always
+      initContainers:
+        - name: "remove-lost-found"
+          image: "busybox:1.32"
+          imagePullPolicy: "IfNotPresent"
+          resources:
+            requests:
+              cpu: 10m
+              memory: 10Mi
+          command: ["rm", "-fr", "/var/lib/mysql/lost+found"]
+          volumeMounts:
+            - name: data
+              mountPath: /var/lib/mysql
+      containers:
+        - name: mysql
+          image: "mysql:5.7.30"
```

Comment:
Wow, that's specific! Why not mysql:5.7 for getting patches?

Reply:
I like to pinpoint on very specific version, to have absolute visibility and control of when and how to upgrade, otherwise a pod might die and come back up with an upgraded version and all of a sudden something might have gotten changed.

Reply:
Sadly MySQL sometimes include breaking changes in the patch versions so it's better to be sure

Reply:
That's true, but bear in mind we've changed the logic a bit. It means we're using mysql which we deploy ourselves in dev mode and use managed database provided by cloud provider for production.